javascript/javascript1/week3/series-duration-of-my-life/series-duration-of-my-life.js

```diff
@@ -0,0 +1,47 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+  {
+    title: "Breaking Bad",
+    days: 16,
+    hours: 21,
+    minutes: 0,
```

Comment:
[The website](https://tiii.me/) shows 2 days 3 hours and 40 minutes. Did you increase it for test purposes?